src/battery-status.tsx

```
@@ -1,33 +1,77 @@
-import { ActionPanel, Detail, List, Action, Icon } from "@raycast/api";
+import { ActionPanel, Detail, List, Action, Icon, showToast, Toast } from "@raycast/api";
```

ESLint

[error] 'List' is defined but never used.

(@typescript-eslint/no-unused-vars)